index.html

```diff
@@ -50,7 +61,7 @@
             </div>
             <div class="col-xs-8">
               <div class="row">
-                <div class="col-xs-12">
+                <div class="col-xs-12" v-if="authorMode&!kioskMode">
```

Comment:
There's a difference between `&` (bitwise) and `&&` (logical) that you should learn. The above should be written:
```html
v-if="authorMode && !kioskMode"
```

Please use spaces around binary operators like `&&`.

See [Bitwise Operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators) and realize that you will rarely use them in day-to-day programming.

See [Logical Operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators) and learn about *short-circuiting*.